crates/bevy_app/src/lib.rs

```diff
@@ -23,7 +24,7 @@ mod plugin;
 mod plugin_group;
 mod schedule_runner;
 mod sub_app;
-#[cfg(not(target_arch = "wasm32"))]
+#[cfg(all(not(target_arch = "wasm32"), feature = "std"))]
 mod terminal_ctrl_c_handler;
```

Comment:
Can't use Control-C in the terminal if your platform doesn't have a terminal...or a keyboard...